DOCUMENTATION.md

```diff
@@ -0,0 +1,150 @@
+<h1 align="center">
+Renovate Configuration
+</h1>
+
+<br>
+
+#### While we already include a broad base configuration by default you can also add to, and extend your own repo's configuration to meet your project's needs. 
+
+<br>
+
+#### By default the base configuration covers most of the common package dependency standards found in popular languages. However if you are interested in customizing further for your language of choice please consult the [Renovate Docs](https://docs.renovatebot.com/). 
+
+<br>
+
+<h2 align="center">
+*** NOTE *** 
+<br>
+***** DO NOT OVERRIDE BASE CONFIGURATIONS LOCALLY *****
+</h2>
+
+  If your team feels the base configuration does not meet your needs please come and contact us in Slack in `#platform-core-services`! We are a lovely bunch and are more than happy to help fix your pain points! We can also explain the reasoning behind the settings chosen and help you find a compromise without sacrificing security. We also welcome PRs against the `cds-snc/renovate-config` repo if you find a bug or have a suggestion for a good base setting other teams might benefit from!
+
+# 
+
+## Table of Contents:
+1. [Base Configuration](#base-configuration)
+2. [Anatomy of a PR](#anatomy-of-a-pr)
+3. [Language specific tips and tricks](#language-specific-tips-and-tricks)
+   1. [Python](#python)
+   2. [Javascript](#javascript-and-nodejs)
+4. [Gotchas](#common-gotchas)
+
+#
+
+<h2 align="center">
+Base Configuration
+</h2>
+
+<br>
+
+### What's included?
+The base configuration covers most popular languages (Python, JS/Node.js, Ruby, Go are all included), sorry Max - no Rust yet! But each configuration setup included looks for best practise dependency tracking files in each project (`package.json` for Node.js, `requirements.txt` for Python projects). This can also be customized locally if your team is using nonconventional names for your tracking files.
+
+<br>
+
+Each repo will also have their own Dependency Dashboard gently lurking in their Issues board on Github. This dashboard shows which updates are waiting to be addressed by developers in the workflow agreed upon by each team (Scrum, Kanban, Sprint etc). When develpers create branches and PRs from the Dependency Dashboard they can then see the most up to date information available to help them integrate the updates. 
+
+<br>
+
+This Dashboard is updated on a weekly basis so that teams don't feel overwhelmed by a constant and neverending list of dependency PRs!
```

Comment:
I think it updates on every run of the app - I'm curious what Renovate support says.

Reply:
Still haven't heard back. I'll leave this part out until we hear otherwise.